biz.aQute.junit.platform.reporting/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
You need to make sure all new projects have the same `.settings/` folder contents as the other projects. Use `biz.aQute.bndlib/.settings/` as a source for copying into the new projects. And code in the new projects should be formatted according to these new settings.

Reply:
Thanks - I originally copied them from another project but it seems that the preferences on master have moved on since I originally created the projects. I'll get this sorted.